### PR TITLE
temp-fork-better-auth

### PR DESCRIPTION
**Pull request by Limerio**

### CODE REVIEW

File: apps/email-service/main.go [hunk 1]
```diff
@@ -31,7 +32,7 @@ func main() {
 	resend_client := resend.NewClient(apiKey)
 	email_server := grpcgoserver.NewEmailServer(resend_client)
 
-	lis, err := net.Listen("tcp", port)
+	lis, err := net.Listen("tcp", fmt.Sprintf("localhost:%s", port))
```
Comment (WookiesRpeople2) [inline apps/email-service/main.go:35]: Local host is already inferred by default don't need to specify it

**Review by WookiesRpeople2 — DISMISSED** [review]

File: apps/email-service/main.go (hunk 1 above)
Comment (Limerio) [reply apps/email-service/main.go:35]: It's not. The code throws an error. If you want to, you can do `:THE_PORT` instead
https://pkg.go.dev/net#Listen

**Review by WookiesRpeople2 — APPROVED** [review]

File: apps/email-service/main.go (hunk 1 above)
Comment (WookiesRpeople2) [reply apps/email-service/main.go:35]: That's so weird for me it worked and in the docs it says that the recommended way is
`ln, err := net.Listen("tcp", ":8080") `

Comment (WookiesRpeople2) [reply apps/email-service/main.go:35]: Ohhh I know why because I used :PORT in the .env directly

**Pull request merged**